src/components/common/homeCard.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from "react";
+import { Card } from "primereact/card";
+
+type Props = {
+  title: string;
+  icon: string;
+};
+
+interface HomeCardProps {
+  title: FC<Props>;
+  value: FC;
+}
+
+const HomeCard: FC<HomeCardProps> = ({ title, value }) => {
```

Comment:
if title and value are strings. then you can re-write in a more simple way.

```
type Props = {
  title: string;
  value: string; 
}
const HomeCard: FC<Props> = ({title, value}) => {
```

Reply:
these props are react components, not strings